src/Extensions/FileFormFactoryExtension.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+namespace SilverStripe\SearchService\Extensions;
+
+use SilverStripe\Assets\Image;
+use SilverStripe\Control\RequestHandler;
+use SilverStripe\Core\Extension;
+use SilverStripe\Forms\CheckboxField;
+use SilverStripe\Forms\DatetimeField;
+use SilverStripe\Forms\Form;
+use SilverStripe\Forms\FormFactory;
+
+class FileFormFactoryExtension extends Extension
```

Comment:
can we make this a more generic `SearchFormFactoryExtension`? Feels like it could apply to any data object form that is used via graphql/js components